### ISSUE TITLE
County-level health equity analysis

### ISSUE THREAD

**Issue by jakefishbein-navapbc** [CDCgov/prime-reportstream#13522]
### Outcome
In order to develop a narrow, high-impact strategy for identifying how we could recruit senders for more equitable data collection, a subset of ELC-funded STLTs will be further investigated to determine geographic distribution of inequity. Indicators of inequality will be used as proxies for identifying specific localities where underrepresentation of particular demographic groups in ELR data may be more pronounced.

### Process
- [x] Conduct county level analysis including ~~the same indicators as the larger analysis~~ broadband access, child poverty, premature death, pcp rate, and uninsured rate. Public health funding by state will also be included.
- [x] Compare county level analysis for ELC-funded cities
- [x] Identify the 25% of counties/cities with the highest combined disparity score
- [ ] Stretch: identify conditions within these locations that are commonly associated with disparate outcomes.
- [x] Meet with engagement leads and Glenn to determine which of those locations we might prioritze based on existing strategy, relationships or other factors
- [x] Identify <5 locations to prioritize for direct outreach

**STLTs**
- Mississippi
- Texas
- Louisiana
- Alabama
- New Mexico
- Oklahoma
- Arkansas
- California
- Wyoming
- Georgia
- Cities will be evaluated along side counties once that data is organized
- Territories will be attended to in a separate ticket.

### Acceptance Criteria
- [x] Locality data analysis
- [x] Report of findings
- [x] Confirmed with the appropriate stakeholders

**Comment by jakefishbein-navapbc** [comment]
@jillian-hammer @mxk0 @chris-kuryak Here is a link to the [WIP data analysis](https://docs.google.com/spreadsheets/d/1Qp9HxAGuXn4rCeeshRjm4H5X4x6ZepZBtyRC-N_uVEE/edit?usp=sharing). I've updated the indicators to be better aligned with the county level scale as there is typically less diversity at this level. The AC has been updated with the change. I may need to make a few other adjustments based on what I do and don't find, and will let you know if this is the case.
